app/models/preserved_copy.rb

```diff
@@ -5,8 +5,7 @@ class PreservedCopy < ApplicationRecord
   INVALID_MOAB_STATUS = 'invalid_moab'.freeze
   INVALID_CHECKSUM_STATUS = 'invalid_checksum'.freeze
   ONLINE_MOAB_NOT_FOUND_STATUS = 'online_moab_not_found'.freeze
-  EXPECTED_VERS_NOT_FOUND_ON_STORAGE_STATUS = 'expected_vers_not_found_on_storage'.freeze
-  FIXITY_CHECK_FAILED_STATUS = 'fixity_check_failed'.freeze
+  UNEXPECTED_VERSION_ON_STORAGE_STATUS = 'unexpected_version_on_storage'.freeze
```

Comment:
i'm guessing we will want to report this to workflow errors?

Reply:
yup, but i believe that'll still be handled whenever the corresponding `AuditResults::UNEXPECTED_VERSION` result code gets processed, which should remain unchanged by this PR.  this was just a rename of the constant and the string that we use for the PresCopy status that reflects this situation.